src/main/java/domain/name/Name.java

```diff
@@ -0,0 +1,23 @@
+package domain.name;
+
+public class Name {
+    private String value;
+    private static final Integer LENGTH_LIMIT = 5;
```

Comment:
```suggestion
    private static final Integer LENGTH_LIMIT = 5;
    
    private final String value;
```
1. 상수는 가장 위에 두는 것이 기본 컨벤션입니다.
2. 이름은 불변값이어야 할 것 같아요.